components/script_bindings/codegen/codegen.py

```diff
@@ -7966,8 +8005,8 @@ def argument_type(descriptorProvider, ty, optional=False, defaultValue=None, var
     return declType.define()
 
 
-def method_arguments(descriptorProvider, returnType, arguments, passJSBits=True, trailing=None,
-                     inRealm=False, canGc=False):
+def method_arguments(descriptorProvider: DescriptorProvider, returnType, arguments, passJSBits: bool = True, trailing=None,
```

Comment:
`returnType` and `arguments` are still untyped. is it possible to add more specific types?